.git-blame-ignore-revs

```diff
@@ -0,0 +1,8 @@
+# Let pre-commit format the code
+a3e059110584c0ed44a707e855df6adea5f70c78
+
+# Replace GUINAME with its value
+3d96f2a209612bed70d65127b5e91c498ca0af0a
+
+# Replace LIBNAME with its value
+70089ad298cede56869763aba105ae757a228e03
```

Comment:
While you are at it, we did some major reformatting when we first set up clang-format and autopep8. Maybe you could locate these commits in the history and add them here?